ci-operator/config/openshift/oc/openshift-oc-master__okd-scos.yaml

```diff
@@ -43,10 +47,10 @@ images:
   dockerfile_path: images/cli-artifacts/Dockerfile.rhel
   from: cli
   inputs:
-    ocp_builder_rhel-8-golang-1.20-openshift-4.16:
+    ocp_builder_rhel-8-golang-1.21-openshift-4.16:
```

Comment:
Bumping to go 1.21 makes sense. However, we have switched to RHEL9 entirely https://github.com/openshift/oc/pull/1752 and I think, this configuration files should be simplified by only having rhel9-1-21 builder and removing the rest. Something similar to https://github.com/openshift/release/pull/51610